tests/ra_aid/database/test_migrations.py

```
@@ -83,276 +80,294 @@ def temp_migrations_dir(temp_dir):
+            f"Initialized migration router with table: {MIGRATIONS_TABLE}"
+        )
+
     def test_ensure_migrations_dir(self, cleanup_db, temp_dir, mock_logger):
         """Test _ensure_migrations_dir creates directory if it doesn't exist."""
         # Set up test paths
         db_path = os.path.join(temp_dir, "test.db")
         migrations_dir = os.path.join(temp_dir, "nonexistent_dir", MIGRATIONS_DIRNAME)
-        
+
         # Initialize manager
         manager = MigrationManager(db_path=db_path, migrations_dir=migrations_dir)
```

Ruff (0.8.2)

Local variable `manager` is assigned to but never used

Remove assignment to unused variable `manager`

(F841)